lib/request.js

```diff
@@ -88,7 +88,7 @@ const request = (profile, userAgent, opt, data) => {
 	.then((b) => {
 		if (DEBUG) console.error(JSON.stringify(b))
 
-		if (b.err) {
+		if (b.err && b.err !== 'OK') {
```

Comment:
Is this correct here? Just asking because as this doesn't seem directly related to the scope of the PR I want to make sure this is an intended change (since it affects all requests)? 😄

Reply:
This is part of the switch to the `1.21` protocol, so yes, intended. From what I can tell, it doesn't break endpoints with older versions.